### DISCUSSION TITLE
Future of this project

### DISCUSSION THREAD

**Discussion by Sharcoux** [Hopding/pdf-lib#1631] [General]
Hi guys,

This project is on hold and I can't reach Hopding. We're actively maintaining a fork at [cantoo/pdf-lib](https://github.com/cantoo-scribe/pdf-lib) where we added document encryption and decryption, and support for drawing svg. We also merged most of the PR opened in this project.

The problem is that it is complex to maintain an open source project when users firstly arrive here, and document their issues or open PRs in this project. Unfortunately, as long as I can't reach Hopding, I don't see what we can do.

Does anyone of you have an idea?

**Comment by alistair0adams** [comment]
Have you tried contacting him on LinkedIn? https://www.linkedin.com/in/andrewjdillon/ Last interaction was a month ago, not a big user not insignificant. Happy to try for you.

Separately, do you know if anyone has done anything to allow access to non-interactive forms? Form XObjects. I get a form filled signed pdf and I want to pull the name of the person who signed it and that appears to be in an XObject. I can get it from the Python pdfreader package but I need JavaScript.

**Reply by Sharcoux** [reply]
Good idea, I've sent a message through there. We'll see.

**Reply by Sharcoux** [reply]
No answer so far...

**Reply by shadoath** [reply]
Seems like he dropped off the map after joining GitHub. His linkedIn is now a 404.

**Comment by nayounsang** [comment]
Hi. My new project is using this library.
I just sent Hopding an email.
I mentioned this discussion and hope to receive an answer.
Until then, I would also like to contribute to the library in your fork.
How about discord? 
Hopding#3760

**Reply by Sharcoux** [reply]
Be my guest.

About Discord, for now I don't think it's necessary. The Issue and PR sections of the pdf-lib fork should be enough, don't you think?

**Reply by nayounsang** [reply]
Ah, Discord is contact if maintainer not see linkdin or email

**Reply by zulander1** [reply]
@Sharcoux, any luck with this? It seems like the maintainer has left, and hasn’t committed anything in almost 5 years.

**Reply by Sharcoux** [reply]
We took over the maintenance. You can use the fork.
